concent_api/core/tests/utils.py

```diff
@@ -616,3 +616,10 @@ def _assert_stored_message_counter_not_increased(self):
 
     def _assert_client_count_is_equal(self, count):
         self.assertEqual(Client.objects.count(), count)
+
+    # TODO: Merge with '_sign_message' after authentication is merged
+    def _get_signature(self, golem_message, priv_key, pub_key):  # pylint: disable=no-self-use
+        golem_message.sig = None
+        serialized_message = dump(golem_message, priv_key, pub_key)
+        loaded_message = load(serialized_message, priv_key, pub_key, check_time = False)
+        return loaded_message.sig
```

Comment:
You don't have to `load()` and `dump()`. That's an overkill. golem-messages has functions for computing the signature.